### DISCUSSION TITLE
As an Administrator I want to use the Central Server and Security Server UIs in Turkmen language so that I can use my preferred language.

### DISCUSSION THREAD

**Discussion by Bibigozel** [nordic-institute/X-Road#2491] [Ideas]
as X-Road is deployed in Turkmenistan local administrators would find it more convenient to use X-Road components in their native language. 
We are prepared to submit the relevant pull request to fulfill this enhancement and provide later updates on X-Road new releases.

**Comment by petkivim** [comment]
Thank you for the heads up regarding your upcoming pull request. We appreciate your contribution. In order to ensure a smooth contribution process, please review the Contributor License Agreement (CLA) and initiate the CLA signing process by following the instructions available here:

https://github.com/nordic-institute/X-Road/blob/develop/CONTRIBUTING.md#sign-the-cla

When opening the pull request on GitHub, please use issue ID `XRDDEV-2811` as a reference. It's a placeholder issue for your contribution:

https://nordic-institute.atlassian.net/browse/XRDDEV-2811

Detailed instructions for adding support for a new UI language are available here:

https://nordic-institute.atlassian.net/wiki/spaces/XRDKB/pages/864288788/How+to+Add+a+New+Language+to+the+Central+Server+or+Security+Server+UI
